lib/factory.ts

```diff
@@ -11,8 +11,10 @@ export interface AnyFactories {
   [key: string]: Factory<any>;
 }
 
+const SEQUENCE_START_VALUE: number = 1;
```

Comment:
The `number` type actually makes this type looser, since it is otherwise typed as a const `1`.

```suggestion
const SEQUENCE_START_VALUE = 1;
```